photon-ml/src/integTest/scala/com/linkedin/photon/ml/BootstrapTrainingTest.scala

```diff
@@ -63,12 +65,12 @@ class BootstrapTrainingIntegTest extends SparkTestUtils {
       drawSampleFromNumericallyBenignDenseFeaturesForLinearRegressionLocal(
         seed.toInt,
         numSamples,
-        BootstrapTrainingTest.NUM_DIMENSIONS)
+        numDimensions)
         .toSeq)
       .map(x => new LabeledPoint(x._1, x._2))
 
     val result: Map[Double, Map[String, Any]] = BootstrapTraining.bootstrap[LinearRegressionModel](
-      BootstrapTrainingTest.NUM_SAMPLES,
```

Comment:
Was using NUM_SAMPLES from the unit test, AND has a local numSamples! Removed the dependency on the unit test, and simplified to have only a local numSamples.